nixos/modules/programs/regreet.nix

```diff
@@ -50,7 +50,7 @@ in
   config = lib.mkIf cfg.enable {
     services.greetd = {
       enable = lib.mkDefault true;
-      settings.default_session.command = lib.mkDefault "${lib.getExe pkgs.cage} -s -- ${lib.getExe cfg.package}";
+      settings.default_session.command = lib.mkDefault "dbus-run-session ${lib.getExe pkgs.cage} -s -- ${lib.getExe cfg.package}";
```

Comment:
are we certain that dbus-run-session is in the PATH?

Reply:
I'm pretty sure it comes with dbus, and that's enabled by default.
Though I'm not sure if it's equivalent to this command in my config https://github.com/fufexan/dotfiles/blob/main/modules/greetd.nix#L12.

Reply:
According to the Arch wiki, "dbus is pulled and installed as a dependency of systemd and user session bus is started automatically for each user" (https://wiki.archlinux.org/title/D-Bus). Since NixOS is a systemd distro, I believe it is safe to assume that dbus is in the users path. On non-systemd distributions that is not necessarily the case however.

Reply:
> According to the Arch wiki, "dbus is pulled and installed as a dependency of systemd and user session bus is started automatically for each user" (https://wiki.archlinux.org/title/D-Bus). Since NixOS is a systemd distro, I believe it is safe to assume that dbus is in the users path. On non-systemd distributions that is not necessarily the case however.

I mean, yes; but it's possible this path does not enjoy the user `$PATH`, just checking.
If you test the change it seems to work, I am fine though.